docs/chiadatalayer.md

```diff
@@ -105,38 +105,46 @@ spec:
 
 If you use a hostPath volume in a kubernetes cluster with multiple nodes, make sure you have set the proper nodeSelector to ensure the data_layer Pod only runs on the node with that hostPath.
 
-In all three cases, the volume will be automatically mounted in the chia container at `/datalayer/server`. You can change the mount path for the server files in the fileserver container though, see the `Fileserver Configuration` section for information on that.
+In all three cases, the volume will be automatically mounted in the chia container at `/datalayer/server`.
 
 See the [Storage](storage.md) documentation for information on CHIA_ROOT persistence, which is separate from the server files volume.
 
 ## Fileserver Configuration
 
-The ChiaDataLayer can optionally run a fileserver sidecar container to serve the data_layer server files. This is disabled by default but can be enabled with the following configuration:
+The ChiaDataLayer can optionally run a fileserver sidecar container to serve the data_layer server files. This will be needed if you plan to mirror data or publish your own data on DataLayer. This is disabled by default but can be enabled with these fileserver configuration options:
 
 ```yaml
 spec:
   fileserver:
     enabled: true
-    # Optional: defaults to the official chia image using data_layer_http. 
-    # But a custom image for the fileserver can be specified here.
+    # Optional: defaults to the official chia image using data_layer_http,
+    # but a custom image for the fileserver can be specified here.
     image: "custom/fileserver:tag"
-    # Optional custom mount path for server files
-    # This is the mount path in the fileserver container only. 
+    # Optional: custom mount path for server files (.dat files)
+    # This is the mount path in the fileserver container only.
     # It will not change the mount path in the chia container.
+    # This path is what webserver software would set the webroot to
     serverFileMountpath: "/custom/path"
-    # Optional custom container port
+    # Optional: custom container port. This port will be
+    # exposed on at a service endpoint on port 80 by default
+    # unless overridden below in the "service" section
```

Comment:
This still points to changing the service port. You can only change the container port the way it's currently written. I think this should say something like `Optional: custom container port. This port defaults to 8575 and only needs to be changed if using a custom webserver that binds to a different port.

Reply:
```suggestion
    # Optional: custom container port. This port defaults to
    # 8575 and only needs to be changed if using  a custom
    # web server that binds to another port.
```

Reply:
Updated with new language